### ISSUE TITLE
A Small (useless) edits for a t-shirt.

### ISSUE THREAD

**Issue by dispherical** [shitoberfest/spam-pullrequests#3]
Original: @Benricheson101

https://github.com/Benricheson101/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/pull/89

**Comment by hacktoberfest-parody** [comment]
@aboutDavid 
Yeah like as if we don't even know that it's  **[C++](https://github.com/Benricheson101/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/pull/89/files)** 
I am like, wow thanks for informing 🤣 
![image](https://user-images.githubusercontent.com/72258330/95093638-9db63000-0746-11eb-9545-19783823b39c.png)
